t/begin.t

```diff
@@ -18,7 +19,13 @@ ok 1;
 use Pod::Simple::DumpAsXML;
 use Pod::Simple::XMLOutStream;
 print "# Pod::Simple version $Pod::Simple::VERSION\n";
-sub e ($$) { Pod::Simple::DumpAsXML->_duo(@_) }
+
+BEGIN {
+  require FindBin;
```

Comment:
Is this necessary? Surely a simple `use lib 't/lib'; use helpers;` should be fine here.